lectures/getting_started_julia/julia_essentials.md

```diff
@@ -1190,7 +1192,7 @@ In particular, write a function `linapprox` that takes as arguments
 * A function `f` mapping some interval $[a, b]$ into $\mathbb R$.
 * two scalars `a` and `b` providing the limits of this interval.
 * An integer `n` determining the number of grid points.
-* A number `x` satisfying `a ≤ x ≤ b`.
+* A number `x` satisfying `a <= x <= b`.
```

Comment:
I think this probably should go back to  `a ≤ x ≤ b` since we want unicode and it isn't in a text block.